README.md

```diff
@@ -91,7 +91,10 @@ output {
    }            
 }               
 ```
+### ecs_compatibility
+ECS compatibility for V8 was added in 1.3.0. For more details on ECS support refer to this [documentation](docs/ecs_compatibility.md).
 
+## Detailed Documentation   
```

Comment:
This section seems out of place as detailed documentation and can be removed. If you still want to include it, this can be added as a link [Ship events to OpenSearch](https://opensearch.org/docs/latest/clients/logstash/ship-to-opensearch/#opensearch-output-plugin) under Project resources